package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -357,6 +357,13 @@ def __init__(
                 )
             )
 
+        n_residues = len(ag.residues)
+        MIN_RESIDUES = 5
+        if n_residues < MIN_RESIDUES:
+            raise ValueError(
+                f"DSSP requires at least {MIN_RESIDUES} residues for secondary structure analysis, but only {n_residues} residue(s) were provided in the selection."
+            )
```

Comment:
```suggestion
        if len(ag.residues) < 6:
            raise ValueError(
                "DSSP requires at least 6 residues for secondary structure analysis, "
                f"but only {len(ag.residues)} residue(s) were provided in the selection."
            )
```

See the original issue "fewer than 6 residues" not 5.
Also there's no need to define one time use variables.

Reply:
ok, doing the changes!

Reply:
done